supervisor/utils/common.py

```diff
@@ -15,14 +15,12 @@
 _DEFAULT: Dict[str, Any] = {}
 
 
-def find_one_filetype(
-    path: Path, filename: str, filetypes: List[str]
-) -> Optional[Path]:
+def find_one_filetype(path: Path, filename: str, filetypes: List[str]) -> Path:
     """Find first file matching filetypes."""
     for file in path.glob(f"**/{filename}.*"):
         if file.suffix in filetypes:
             return file
-    return None
+    raise ConfigurationFileError(f"{path!s}/{filename}.({filetypes}) not exists!")
```

Comment:
```suggestion
    raise ConfigurationFileError
```
Value is not used

Reply:
all or nothing. I can remove all Exception("message") or we going this style now